mycore-user2/src/main/java/org/mycore/user2/MCRUserServlet.java

```diff
@@ -421,7 +427,7 @@ private void updateBasicUserInfo(Element u, MCRUser user) {
             .map(attributes -> attributes.getChildren("attribute"))
             .orElse(Collections.emptyList());
         Set<MCRUserAttribute> newAttrs = attributeList.stream()
-            .map(a -> new MCRUserAttribute(a.getAttributeValue("name"), a.getAttributeValue("value")))
+            .map(a -> new MCRUserAttribute(a.getAttributeValue(ATTRIBUTE_NAME), a.getAttributeValue("value")))
```

Comment:
add `ATTRIBUTE_VALUE` for reasons of consistency

Reply:
We want the value of the attribute "name" here. So ATTRIBUTE_NAME is correct.
For example in the same line: a.getAttributeValue("value") -> its ATTRIBUTE_VALUE. And not ATTRIBUTE_VALUE_VALUE.